packages/bundler/src/types.ts

```diff
@@ -0,0 +1,10 @@
+import { RollupWarning, RollupError } from 'rollup';
+
+export type BundlerError = RollupError;
+export type BundlerWarning = RollupWarning;
+
+export type BundlerMessage =
+  | { kind: 'START' }
+  | { kind: 'UPDATE' }
+  | { kind: 'WARN'; warning: BundlerWarning }
+  | { kind: 'ERROR'; error: BundlerError }
```

Comment:
Our convention elsewhere in the codebase is to use `type` and lowercase for message discriminators. e.g. `{type: 'start' } 

Maybe not the best choice, but better to me consistent and then change the convention everywhere in one go.

Reply:
I have changed it to `type`, I do believe `kind` is the convention and `type` is a redux thing.  

I could definitely be wrong and it does not matter too much and as you say, it should be consistent.